docs/tctl/workflow/reset.md

```diff
@@ -44,8 +49,8 @@ tctl workflow reset --run_id <id>
 
 ### `--event_id`
 
-Specify the `eventId` of any event after `WorkflowTaskStarted` to which you want to reset.
-Valid values are `WorkflowTaskCompleted`, `WorkflowTaskFailed`, and `WorkflowTaskTimeout`.
+Specify the `eventId` number of the event to which you want to reset.
+The event type must be one of: `WorkflowTaskCompleted`, `WorkflowTaskFailed`, and `WorkflowTaskTimeout`.
```

Comment:
Clarifying that `WorkflowTaskCompleted` is not a valid value for `--event_id`. Must be a number.